src/api/cpp/cvc5.h

```diff
@@ -4750,10 +4750,12 @@ class CVC5_EXPORT Solver
    *     (check-synth)
    * \endverbatim
    *
-   * @return the result of the check, which is unsat if the check succeeded,
-   * in which case solutions are available via getSynthSolutions.
+   * @return the result of the check, which is "solution" if the check found a
```

Comment:
Can you format this so that the `@return` is free standing and text for it is a block right beside it (like you did in Python)?